### ISSUE TITLE
custom_transformer.yaml file

### ISSUE THREAD

**Issue by Riretta** [CompVis/taming-transformers#119]
I try using a custom_transformer.yaml but I get error on the cond_stage_key can you explain to me how to create a correct file for transformer phase?

please help me understand

**Comment by Blackkinggg** [comment]
I have the same question. Help###

**Comment by sdulyq** [comment]
how to solve it? please

**Comment by Riretta** [comment]
I don't know how I solve that,
In my custon_transformer.yaml I set:
model:
  base_learning_rate: 4.5e-06
  target: taming.models.cond_transformer.Net2NetTransformer
  params:
    transformer_config:
      target: taming.modules.transformer.mingpt.GPT
      params:
        vocab_size: 1024
        block_size: 512
        n_layer: 24
        n_head: 16
        n_embd: 1024
    first_stage_config:
      target: taming.models.vqgan_caps.VQModel
      params:
        ckpt_path: logs/2021-11-17T17-23-22_custom_vqgan/checkpoints/last.ckpt
        embed_dim: 256
        n_embed: 1024
        ddconfig:
          double_z: false
          z_channels: 256
          resolution: 256
          in_channels: 3
          out_ch: 3
          ch: 128
          ch_mult:
          - 1
          - 1
          - 2
          - 2
          - 4
          num_res_blocks: 2
          attn_resolutions:
          - 16
          dropout: 0.0
        lossconfig:
          target: taming.modules.losses.DummyLoss
    cond_stage_config:  __is_unconditional__


I don't know if it helps
